bin/mozilla-en-US.py

```diff
@@ -115,15 +117,23 @@ def push(path):
 # Clone or update source repository
 url = 'https://hg.mozilla.org/l10n/gecko-strings/'
 target = 'source'
-pull(url, target)
+code = pull(url, target)
+
+# In case of a failure, quit early (bug 1475603)
+if code != 0:
+    quit()
 
 for repo in TARGET_REPOS.keys():
     ending = repo + '-central'
     url = 'ssh://hg.mozilla.org/users/m_owca.info/' + ending
     target = os.path.join('target', ending)
 
     # Clone or update target repository
-    pull(url, target)
+    code = pull(url, target)
+
+    # In case of a failure, continue to next repo
+    if code != 0:
+        continue
```

Comment:
I'd also `quit(code)` here.

That way, we should get nasty warnings in logs, and the offending repo is coming last in the log.